bridge/opentracing/bridge_test.go

```diff
@@ -660,3 +721,54 @@ func TestBridgeCarrierBaggagePropagation(t *testing.T) {
 		}
 	}
 }
+
+func TestBridgeFiledEncoder(t *testing.T) {
```

Comment:
Please add test cases for:

- `func (e *bridgeFieldEncoder) EmitInt32(key string, value int32)`
- `func (e *bridgeFieldEncoder) EmitInt64(key string, value int64)`
- `func (e *bridgeFieldEncoder) EmitUint32(key string, value uint32)`
- `func (e *bridgeFieldEncoder) EmitUint64(key string, value uint64)`